src/lib/table/table.md

```diff
@@ -278,3 +278,50 @@ Table's default role is `grid`, and it can be changed to `treegrid` through `rol
 
 `mat-table` does not manage any focus/keyboard interaction on its own. Users can add desired 
 focus/keyboard interactions in their application.
+
+### Using flex layout
```

Comment:
Reword this to be something like `` Tables with `display: flex `` so that it's not confused with `@angular/flex-layout`